src/schema/users.ts

```diff
@@ -439,6 +450,11 @@ export const typeDefs = /* GraphQL */ `
     Returns the latest top reader badge for the user
     """
     topReader: UserTopReader
+
+    """
+    Flags for the user
+    """
+    flags: UserFlags
```

Comment:
You don't need this really for the feature since you will look for the value in boot? But if you want to expose this you have to adjust `graphorm/index`, look for `flags` definition for `Settings` entity